group/add-member/schema.json

```diff
@@ -56,66 +57,5 @@
       }
     }
   },
-  "additionalProperties": false,
```

Comment:
:fire: if someone imports this file without those definitions, it makes no sense

Reply:
By which I mean - I deliberately made JSON files so that were complete, so that other languages which can read JSON-Schema, can directly take this file and use it. Trying to make it easier for Go and Rust implementations to "consume" the spec

Reply:
yeah i wanted to use native json schema refs initially but that turned out to be horrible https://github.com/ssbc/ssb-tribes2/issues/27#issuecomment-1297301592

but a build step might make sense

Reply:
moved these to partialSchema.json's and now we generate the complete usable schema.json's